src/io/flutter/utils/FlutterModuleUtils.java

```diff
@@ -59,21 +59,24 @@ public static boolean hasFlutterModule(@NotNull Project project) {
     // If not IntelliJ, assume a small IDE (no multi-module project support).
     // Look for a module with a flutter-like file structure.
     if (!PlatformUtils.isIntelliJ()) {
-      for (Module module : ModuleManager.getInstance(project).getModules()) {
-        if (usesFlutter(module)) {
-          return true;
-        }
+      if (CollectionUtils.anyMatch(getModules(project), FlutterModuleUtils::usesFlutter)) {
```

Comment:
As a style thing, I'm not convinced this is an improvement? A simple for-each loop in a small function is self-contained and universally understood.

(But I won't block this if others disagree.)